spec/controllers/master/games_controller_spec.rb

```diff
@@ -16,8 +16,8 @@
 
       xhr :get, :index
 
-      result = JSON.parse(response.body)['games'].map {|g| g['id']}
-      expected_result = games.map {|g| g.id.to_s}
+      result = JSON.parse(response.body)['games'].map {|g| g['id']}.sort
```

Comment:
Space between { and | missing.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.